_docs/setup/kubernetes/ansible-install.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Installing with Ansible
+overview: Instructions on using the included Ansible playbook to perform installation.
+
+order: 40
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+The Ansible scenario defined within this project will allow you to : 
+
+- Install `Istio` Distribution and set the path of the `istioctl` go client (if you execute the command manually)
+- Deploy Istio on Openshift or Kubernetes by specifying different parameters (version, enable auth, deploy bookinfo, ...)
+- Specify the addons to be deployed such as `Grafana`, `Prometheus`, `Servicegraph`, `Zipkin` or `Jaeger`
+
+## Prerequisites
+
+- [Ansible 2.4](http://docs.ansible.com/ansible/latest/intro_installation.html)
+
+Refer to the Ansible Installation Doc on how to install Ansible on your machine.
+To use [Minishift](https://docs.openshift.org/latest/minishift/command-ref/minishift_start.html) or [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) for local clusters, please refer to their respective documentation. 
+
+## Execution
+
+The role assumes that the user :
```

Comment:
Can you put line 28/29/31 in the prerequisites to be consistent with our other documentation?